app/services/create_proposed_organisation_edit.rb

```diff
@@ -36,6 +36,7 @@ def merge_in_non_published_fields
 
   def send_email_to_superadmin_about_org_edit
     superadmin_emails = user_klass.superadmins.pluck(:email)
```

Comment:
@decareano yes, the presence of the `receive_message_chain` indicates that our object is tightly coupled to the set of related objects; which leads us to refactor the messages that get sent between the objects to be simpler.

Have you heard of the law of demeter?  https://en.wikipedia.org/wiki/Law_of_Demeter that's what we're noticing here - that the objects know too much about each other